src/views/Conference.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="conference-page"
+    >
+      <div
+        id="jitsi"
+      />
+    </div>
+  </Page>
+</template>
+<script>
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why this rule should be disabled here?

Reply:
If I define it with variable I will get other error - that it's not used.

Reply:
That rule means that I can define new class instance without assign a variable.

Reply:
Yes, this is how works `no-new`, but I'm asking about `import/no-extraneous-dependencies`

Reply:
Right, it's outdated. I will remove it.